Routes [GET, POST, PUT, DELETE, PATCH] examples

lakovicj opened the pull request:
'test' URI should be uncommented in VerifyCsrfToken.php middleware.

lakovicj committed:
GET, POST, PUT, DELETE, PATCH examples. In Http/Middleware/VerifyCsrfToken.php, 'test' URI should be uncommented for routes to work properly.

dejan-stokic commented on laravel-example/routes/web.php:
```diff
@@ -16,3 +16,23 @@
+
+Route::DELETE('/test', function() {
+    echo 'Test DELETE';
+});
```
@lakovicj Dodaj neki primer sa parametrima koje ces da prikazes u response-u. 
Treba da name-ujes svaku rutu.
Nemoj da koristis _echo_ neka svaki primer ima regularne povratne vrednosti.

dejan-stokic commented:
@lakovicj Prefiksuj svaki komit sa id-jem taska za koji je vezan. Npr. JLLR-15: Routes examples.
Bice lakse pronaci na koji task se odnosi i sta treba da obuhvati. Olaksava nam pracenje komiteva.

lakovicj committed:
JLLR-15: Laravel routes FIX

All routes are named now. Routes return message with appropriate HTTP status code. Examples with URL parameters are added.
